src/main/resources/view/ListingListPanel.fxml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
```

Comment:
Are wildcard imports allowed for .fxml classes?

Reply:
Tests did not fail tho but might not be